maps/utk.yml

```
@@ -2345,9 +2328,9 @@ properties:
       sources:
       - iso639-2b
     definition:
-      default: Enter a language used in the resource as text (e.g. English). When
-        multiple languages are present, use multiple fields. If no language is present,
-        enter "No linguistic content."
+      default: Enter a language used in the resource. When multiple languages 
```

Remove extra spaces:

```suggestion
      default: Enter a language used in the resource. When multiple languages
```